portal/static/portal/sass/styles.scss

```diff
@@ -1231,8 +1286,8 @@ td {
       font-weight: 500;
     }
 
-    .button--secondary--dark {
-      @include _margin(0px, 0px, 0px, 30px);
+    .button--primary--general-play {
```

Comment:
In BEM, "button" is your block and "primary" and "general-play" would both be modifiers.  You shouldn't need two modifiers, you can use two different classes for something with more than one property.